crates/biome_js_analyze/tests/specs/nursery/noStaticElementInteractions/invalid.jsx

```diff
@@ -0,0 +1,100 @@
+	<i onClick={() => {}} />
+	<kbd onClick={() => {}} />
+	<link onClick={() => {}} href="#" />
+	<map onClick={() => {}} />
+	<meta onClick={() => {}} />
+	<noscript onClick={() => {}} />
+	<object onClick={() => {}} />
+	<picture onClick={() => {}} />
+	<q onClick={() => {}} />
+	<rp onClick={() => {}} />
+	<rt onClick={() => {}} />
+	<s onClick={() => {}} />
+	<samp onClick={() => {}} />
+	<script onClick={() => {}} />
+	<section onClick={() => {}} />
+	<small onClick={() => {}} />
+	<source onClick={() => {}} />
+	<span onClick={() => {}} />
+	<style onClick={() => {}} />
+	<title onClick={() => {}} />
+	<track onClick={() => {}} />
+	<u onClick={() => {}} />
+	<var onClick={() => {}} />
+	<wbr onClick={() => {}} />
+	
+	{/* // Handlers */}
+	<div onKeyDown={() => {}} />
+	<div onKeyPress={() => {}} />
+	<div onKeyUp={() => {}} />
+	<div onClick={() => {}} />
+	<div onMouseDown={() => {}} />
+	<div onMouseUp={() => {}} />
+
+	{/* Presentation is a special case role that indicates intentional static semantics */}
+	<div role="presentation" onClick={() => {}} />
+	<div role="presentation" onKeyDown={() => {}} />
+	{/* HTML elements attributed with an abstract role */}
+	<div role="command" onClick={() => {}} />
+	<div role="composite" onClick={() => {}} />
+	<div role="input" onClick={() => {}} />
+	<div role="landmark" onClick={() => {}} />
+	<div role="range" onClick={() => {}} />
+	<div role="roletype" onClick={() => {}} />
+	<div role="sectionhead" onClick={() => {}} />
+	<div role="select" onClick={() => {}} />
+	<div role="structure" onClick={() => {}} />
+	<div role="widget" onClick={() => {}} />
+	<div role="window" onClick={() => { }} />
+	
+	{/* <summary> is inherently an interactive element, but in eslint-plugin-jsx-a11y, 
+	    it was made non-interactive due to the influence of an external library. */}
+	{/* ref: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/0be7ea95f560c6afc6817d381054d914ebd0b2ca/src/util/isInteractiveElement.js#L86-L89 */}
+	{/* <summary onClick={() => {}} /> */}
+	
+	{/* This element is rejected by HTML Standard */}
+	{/* ref: https://lists.w3.org/Archives/Public/public-whatwg-archive/2012Aug/0298.html */}
+	{/* <content onClick={() => {}} /> */}
+
+	{/* This element is rejected by HTML Standard */}
+	{/* ref: https://html.spec.whatwg.org/multipage/obsolete.html */}
+	{/* <acronym onClick={() => {}} /> */}
+	{/* <applet onClick={() => {}} /> */}
+	{/* <frame onClick={() => { }} /> */}
+	{/* <frameset onClick={() => { }} /> */}
+	{/* <center onClick={() => {}} /> */}
+	{/* <font onClick={() => {}} /> */}	
+	{/* <big onClick={() => {}} /> */}
+	{/* <blink onClick={() => {}} /> */}
+	{/* <rtc onClick={() => {}} /> */}
+	{/* <xmp onClick={() => {}} /> */}
+	{/* <strike onClick={() => {}} /> */}
+	{/* <param onClick={() => {}} /> */}
+	{/* <keygen onClick={() => {}} /> */}
+	{/* <noembed onClick={() => {}} /> */}
+	{/* <spacer onClick={() => {}} /> */}
+	{/* <tt onClick={() => {}} /> */}
```

Comment:
Are these commented on purpose? If so, we should add a comment saying why, if not, we should remove them or uncomment them.